atom/browser/native_window.cc

```diff
@@ -434,6 +434,10 @@ void NativeWindow::OverrideWebkitPrefs(content::WebPreferences* prefs) {
     prefs->experimental_webgl_enabled = b;
   if (web_preferences_.Get("webaudio", &b))
     prefs->webaudio_enabled = b;
+  if (web_preferences_.Get("allow-displaying-insecure-content", &b))
```

Comment:
We should move this bellow the `web-security` lines, otherwise these options will always be overwritten by `web-security`. It is very likely that people would like to change `allow-displaying-insecure-content` when they have a custom `web-security` setting.